massa-network-worker/src/peer_info_database.rs

```diff
@@ -336,6 +336,76 @@ impl PeerInfoDatabase {
         Ok(())
     }
 
+    pub async fn whitelist(&mut self, ips: Vec<IpAddr>) -> Result<(), NetworkError> {
+        for ip in ips.into_iter() {
+            let old_pt = if let Some(peer) = self.peers.get_mut(&ip) {
+                let pt = peer.peer_type;
+                if pt == PeerType::WhiteListed {
+                    continue;
+                }
+                peer.peer_type = PeerType::WhiteListed;
+                pt
+            } else {
+                let mut p = PeerInfo::new(ip, false);
+                p.peer_type = PeerType::WhiteListed;
+                self.peers.insert(ip, p);
+                continue;
```

Comment:
I don't understand why you continue here. You don't want to call : `self.increase_global_active_out_connection_attempt_count(PeerType::WhiteListed, &ip)?` to increase the count now that there is a new whitelist peer ?

Reply:
Because we didn't know this peer so there is no connection counter to update